roles/php/tasks/macos/install-php.yml

```diff
@@ -56,4 +56,17 @@
     src: "macos/valet-sh.conf.j2"
     dest: "{{ valet_sh_prefix }}/etc/{{ current_php }}/conf.d/95-valet-sh.ini"
     mode: '0644'
-  notify: "brew | {{ current_php }} | restarted"
\ No newline at end of file
+  notify: "brew | {{ current_php }} | restarted"
+
+- name: "create symlink for default php"
+  file:
+    src: "/usr/local/bin/php7.3"
```

Comment:
will this always be php7.3?

Reply:
Only set for the "install" process. The php version will be changed automatically after the "install" command to your local default version

Reply:
have thought about it. the symlink is now only created if it does not exist